app/assets/stylesheets/pages.scss

```diff
@@ -706,6 +706,9 @@
 			background-color: $white;
 			width: 100%;
 		}
+	}
+	.dsc-info {
```

Comment:
Does this do anything? Looks like it is immediately closed.

Reply:
Thanks, no.